client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -145,6 +145,10 @@ class AutoRenewToggle extends Component {
 	renderTextStatus() {
 		const { translate, isEnabled } = this.props;
 
+		if ( this.isUpdatingAutoRenew() ) {
+			return translate( 'Auto-renew (…)' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Enable Auto Renew' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-status-auto-renew-toggling-text)